src/test/java/net/minestom/server/entity/PlayerHeldIntegrationTest.java

```diff
@@ -45,7 +45,7 @@ public void playerHeldEvent(Env env) {
         var listener = env.listen(PlayerChangeHeldSlotEvent.class);
         listener.followup(event -> {
             assertEquals(player, event.getPlayer());
-            assertEquals(1, event.getSlot());
+            assertEquals(1, event.getNewSlot());
```

Comment:
You probably also want to add an assert for the getOldSlot method

Reply:
Added that along with a slightly more complicated test